acceptance/helpers/machine/apps.go

```diff
@@ -18,11 +18,9 @@ func (m *Machine) MakeApp(appName string, instances int, deployFromCurrentDir bo
 }
 
 func (m *Machine) MakeContainerImageApp(appName string, instances int, containerImageURL string) string {
-	pushOutput, err := m.Epinio("", "apps", "push",
-		"--name", appName,
+	pushOutput, err := m.Epinio("", "apps", "push", appName,
```

Comment:
Should this be EpinioPushDir ?

Reply:
This is also used to push container imagers.